src/Components/Header/Header.css

```diff
@@ -0,0 +1,132 @@
+:root{
+    --text-gray-dark :rgb(216, 216, 216);
+    --bg-sidebar :  #161618;
+    --bg-sidebar-hover : rgb(46, 46, 46);
+}
+nav{
+    display: flex;
+    background-color: #161618;
```

Comment:
Here as well. Please use color variables

Reply:
Thanks for point it out